sections/04_study.md

```diff
@@ -188,139 +188,91 @@ disease, which will lead to novel diagnostics and therapeutics.
-also important to understand how these computational models make their
-predictions. To handle this, several papers have focused on understanding
-machine learning models [@tag:Alipanahi2015_predicting @tag:Lanchantin2016_motif
-@tag:Shrikumar2016_blackbox]. [@tag:Alipanahi2015_predicting] was the first to
-introduce a visualization method for a deep learning model on the TFBS task, and
-they did so by visualizing the learned convolution filters which were
-informative for the model’s prediction of a specific sample. However, this
-approach was specific to visualizing certain samples fed through their
-particular model. [@tag:Lanchantin2016_motif] introduced a suite of
-state-of-the-art deep learning models and new visualizations techniques for a
-more in-depth analysis of TFBSs. Furthermore, [@tag:Shrikumar2016_blackbox]
-introduced an advanced visualization method and toolbox for analyzing possible
-TFBS sequences. [@tag:Alipanahi2015_predicting] also introduced mutation maps,
-where they could easily mutate, add, or delete base pairs in a sequence and see
-how the model changed its prediction. This is something that would be very time
-consuming in a lab setting, but easy to simulate using their model.
-Visualization techniques on deep learning models are important because they can
-provide new insights on regulatory mechanisms and can lead biologists to test
-and verify in a lab setting, leading to new biomedical knowledge. Since the
-"linguistics" of DNA are unclear, interpretability of models is crucial to
-pushing our understanding forward.
-
-`TODO: cut RNA-binding proteins from above section, refer to representative
-papers`
+large number of genomic sequences. Lanchantin et al. [@tag:Lanchantin2016_motif]
+introduced several new convolutional and recurrent neural network models that
+further improved TFBS predictive accuracy. Due to the motif-driven nature of the
+TFBS task, most architectures have been convolutional-based
+[@tag:Zeng2016_convolutional]. While many models for TFBS prediction resemble
+computer vision and natural language processing (NLP) tasks, it is important to
+note that DNA sequence tasks are fundamentally different than NLP tasks, and
+thus the models should be adapted from traditional deep learning models in order
+to account for such differences. For example, motifs may appear in either strand
+of a DNA sequence, resulting in two different forms of the motif (forward and
+reverse complement) due to complementary base pairing. To handle this issue,
+Shrikumar et al. [@tag:Shrikumar2017_reversecomplement] created a convolutional
+model that can find motifs in both directions. Since deep learning for protein
+binding prediction is still in early stages, we expect to see an increase in
+domain-specific architectures for this task.
+
+Despite these advances, several challenges remain. First, because the input data
+(ChIP-seq measurements) are continous and the output (whether or not there is TF
```

Comment:
Something may have been lost in this contraction.  Can we make it clear that the output is the output of the deep learning system?  Making this binary is a modeling choice, not a necessity (though we have discussed previously why alternatives like regression aren't ideal either).